docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 ---
-title: Getting Started With Access Controls
-description: Get started using Teleport Access Controls.
+title: Getting Started With Role-Based Access Control
+description: Teleport Role-Based Access Control.
 ---
 
-In Teleport, any local, SSO, or robot user can be assigned one or several roles.
-Roles govern access to databases, SSH servers, Kubernetes clusters, Windows
-desktops, and web apps.
+In Teleport, RBAC (Role-Based Access Control) is a system that helps you manage who can access various resources such as databases, SSH servers, Kubernetes clusters, Windows desktops, and web apps. 
 
-We will start with local users and preset roles, assign roles to SSO users, and
-wrap up with creating your own role.
+## What is RBAC in Teleport?
 
-## Prerequisites
+In Teleport, any local, SSO, or robot user can be assigned one or several roles. Roles govern access to your resources and infrastructure.
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+Each role defines specific permissions to ensure that users only have access to what they need. 
 
-- (!docs/pages/includes/tctl.mdx!)
+## What is a role?
 
-(!docs/pages/includes/permission-warning.mdx!)
+Roles determine what resources a user can access and what actions they can perform within the system. 
 
-## Step 1/3. Add local users with preset roles
+By creating and assigning roles, administrators can enforce security policies, manage access, and ensure users have the appropriate level of privileges based on their responsibilities.
 
-Teleport provides several preset roles:
+## Teleport users
 
-(!docs/pages/includes/preset-roles-table.mdx!)
+Users are entities that can authenticate and interact with the resources within a Teleport cluster. These users can have different levels of access depending on the roles assigned to them. 
 
-<Tabs>
-<TabItem scope={["oss"]} label="Teleport Community Edition">
-Invite the local user Alice as cluster `editor`:
+Teleport supports both internal users and external users, which differ mainly in how they authenticate and how they are managed.
 
-```code
-$ tctl users add alice --roles=editor
-```
-</TabItem>
-<TabItem scope={["enterprise", "cloud"]} label="Commercial">
-Invite the local user Alice as cluster `editor` and `reviewer`:
+### Internal users
```

Comment:
I would standardize on "local" and "SSO" users instead of "internal" and "external":

```
grep -hREoi "(internal|external|local|SSO) user" docs/pages | tr '[:upper:]' '[:lower:]'| sort | uniq -c
  13 external user
   1 internal user
 131 local user
  29 sso user
```